### PR TITLE
Increase test timeouts

### PR DESCRIPTION
**Pull request by dsvoronin**
Some gradle test kit tests goes up to 8 min according to statistics
We should tune that, but right now just removing blocker

### CODE REVIEW

File: subprojects/build.gradle.kts [hunk 1]
```diff
@@ -287,7 +287,7 @@ subprojects {
 
                 systemProperty("isTest", true)
 
-                systemProperty("junit.jupiter.execution.timeout.default", MINUTES.toSeconds(5))
+                systemProperty("junit.jupiter.execution.timeout.default", MINUTES.toSeconds(10))
```
Comment (eugene-krivobokov) [inline subprojects/build.gradle.kts:290]: Will you plan investigation?

Comment (dsvoronin) [reply subprojects/build.gradle.kts:290]: created internal MBS-10204 for this

File: subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt [hunk 2]
```diff
@@ -19,16 +19,13 @@ import com.avito.utils.logging.CILogger
...
-import org.junit.jupiter.api.Timeout
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
-import java.util.concurrent.TimeUnit.SECONDS
 
 internal class TestExecutorIntegrationTest {
 
     private val logger = FakeCILogger()
 
-    @Timeout(10, unit = SECONDS)
```
Comment (sergioboi) [inline subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:31]: Why did you remove those?

File: subprojects/build.gradle.kts (hunk 1 above)
Comment (sergioboi) [reply subprojects/build.gradle.kts:290]: Where did you see the statistics?

Comment (dsvoronin) [reply subprojects/build.gradle.kts:290]: it's in task description, i don't have one, just seeing this time to time on Teamcity's "Tests" tab

File: subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt (hunk 2 above)
Comment (dsvoronin) [reply subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:31]: because i believe global timeouts is enough, and we don't need to tune it per test

File: subprojects/build.gradle.kts (hunk 1 above)
Comment (eugene-krivobokov) [reply subprojects/build.gradle.kts:290]: How many tests require 10 min? Can we override timeout only for them?

Comment (dsvoronin) [reply subprojects/build.gradle.kts:290]: i just don't know; seems like it's random with GradleTestKit one's
we didn't have any of timeouts, now added 10 min timeout and planned to add some statistic and fix it
i see it as an improvement worth merging,
because in current state we only introduce flakiness

File: subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt (hunk 2 above)
Comment (sergioboi) [reply subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:31]: I agree in common but in that Test `Timeout` is an intention of our testing we fix timeouts.

Comment (dsvoronin) [reply subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:31]: i found that sometimes this test timings are also fluctuate and could run over 30sec.
I don't know what is the correct value here, so just removed it in favor of global timeouts

Comment (sergioboi) [reply subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:31]: In my opinion, this test should take 1 sec maximum. We could pairly fix this strange behavior

**Review by eugene-krivobokov — APPROVED** [review]

File: subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt [hunk 3]
```diff
@@ -19,16 +19,13 @@ import com.avito.utils.logging.CILogger
 import com.avito.utils.logging.StubCILogger
 import com.avito.utils.logging.commonLogger
 import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.Timeout
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
-import java.util.concurrent.TimeUnit.SECONDS
 
 internal class TestExecutorIntegrationTest {
```
Comment (eugene-krivobokov) [inline subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt:25]: Please, make an issue about testing in minikube/kind/... like environment.
It will be easier to cover edge-cases more hermetically.
